src/citrine/gemtables/rows.py

```diff
@@ -53,10 +53,13 @@ class MaterialRunByTemplate(Serializable['MaterialRunByTemplate'], Row):
 
     templates = properties.List(properties.Object(LinkByUID), "templates")
     typ = properties.String('type', default="material_run_by_template", deserializable=False)
+    tags = properties.Optional(properties.List(properties.String), "tags")
```

Comment:
Could you please add `tags` to the docstring as well. How are the tags used logically?

Reply:
I'm about to push one more change that includes the docs and the docstring! I will tag you. Thanks.